java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -384,6 +384,49 @@ void testMD5HashMixed() {
     }
   }
 
+  @Test
+  void testMD5HashLists() {
+    try (ColumnVector v0 = ColumnVector.fromStrings(
+          "a", "B\n", "dE\"\u0100\t\u0101 \u0500\u0501",
+          "A very long (greater than 128 bytes/char string) to test a multi hash-step data point " +
+          "in the MD5 hash function. This string needed to be longer.",
+          null, null);
+         ColumnVector v1 = ColumnVector.fromStrings(
+           null, "c", "\\Fg2\'",
+           "A 60 character string to test MD5's message padding algorithm",
+           "hiJ\ud720\ud721\ud720\ud721", null);
+         ColumnVector v2 = ColumnVector.fromStrings(
+           "a", "B\nc",  "dE\"\u0100\t\u0101 \u0500\u0501\\Fg2\'",
+           "A very long (greater than 128 bytes/char string) to test a multi hash-step data point " +
+           "in the MD5 hash function. This string needed to be longer.A 60 character string to " +
+           "test MD5's message padding algorithm",
+           "hiJ\ud720\ud721\ud720\ud721", null);
+
+         ColumnVector result01 = ColumnVector.md5Hash(v0, v1);
+         ColumnVector result2 = ColumnVector.md5Hash(v2);
```

Comment:
result2 is not asserted

Reply:
Missed this test, didn't modify it for list testing. Fixing now.